pysnooper/tracer.py

```diff
@@ -133,6 +137,16 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_value, exc_traceback):
         sys.settrace(self.original_trace_function)
 
+    def set_padding(self, thread_info):
+        current_thread_len = len(thread_info)
+        if self.padding < current_thread_len:
+            self.padding = current_thread_len
+            return thread_info
+        else:
+            needed_spaces = " " * (self.padding - current_thread_len)
+            return "{thread_info}{needed_spaces}".format(**locals())
```

Comment:
Try:

```python
self.padding = max(self.padding, current_thread_len)
return thread_info.ljust(self.padding)
```

Reply:
@4383 I agree with Alex's comments, `ljust` is better.